src/utils.ts

```diff
@@ -24,6 +26,23 @@ export function getCheckRunPayloadWithScanResult(trivyStatus: number, dockleStat
     return checkRunPayload;
 }
 
+export function getScanReportPath(): string {
+  const scanReportPath = `${fileHelper.getContainerScanDirectory()}/scanReport.json`;
+  return scanReportPath;
+}
```

Comment:
Why are you exporting this?

Reply:
Also, 2 lines which aren't used in multiple places doesn't warrant a function.

Reply:
Removed the function